flink-connectors/flink-connector-rabbitmq/src/main/java/org/apache/flink/streaming/connectors/rabbitmq/Util.java

```diff
@@ -0,0 +1,17 @@
+package org.apache.flink.streaming.connectors.rabbitmq;
```

Comment:
CI is failing because this file is missing a license header

Reply:
Ah, thank you! Did you find this in the CI logs?

Reply:
Years of experience reading maven output :)
This is the line
```
[ERROR] Failed to execute goal org.apache.rat:apache-rat-plugin:0.12:check (default) on project flink-parent: Too many files with unapproved license: 1 See RAT report in: /__w/2/s/target/rat.txt -> [Help 1]
```

It doesn't tell us the file, but it's obvious when you look at the PR changes.

Reply:
I'll look for that one next time :)